### PR TITLE
Split into modules

### PR DESCRIPTION
**Pull request by JirkaVrbka**

### CODE REVIEW

File: ApiLayer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ApiLayer</artifactId>
+
+    <properties>
+        <maven.compiler.source>14</maven.compiler.source>
+        <maven.compiler.target>14</maven.compiler.target>
+    </properties>
```
Comment (Michal-MK) [inline ApiLayer/pom.xml]: I would stick to 12 as in the original project pom

File: BusinessLayer/pom.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>BusinessLayer</artifactId>
+
+    <properties>
+        <maven.compiler.source>14</maven.compiler.source>
+        <maven.compiler.target>14</maven.compiler.target>
+    </properties>
```
Comment (Michal-MK) [inline BusinessLayer/pom.xml]: I would stick to 12 as in the original project pom

**Review by Michal-MK — COMMENTED** [review]
https://stackoverflow.com/questions/3509690/what-are-best-practices-for-maven-modules-naming

Lower-case and dash separators, I would not use Upper-case anywhere else other than class names
same goes for git, branch names follow the same syntax

possibly name the folders(modules) just as:
- dao
- business
- api

the "Layer" is redundant imo.

File: ApiLayer/pom.xml (hunk 1 above)
Comment (JirkaVrbka) [reply ApiLayer/pom.xml]: changed

File: BusinessLayer/pom.xml (hunk 2 above)
Comment (JirkaVrbka) [reply BusinessLayer/pom.xml]: changed

File: ApiLayer/pom.xml (hunk 1 above)
Comment (JirkaVrbka) [reply ApiLayer/pom.xml]: webhook check

**Review by JirkaVrbka — COMMENTED** [review]
Webhook sample

**Review by Michal-MK — APPROVED** [review]

**Pull request merged**